src/qililab/calibration/calibration_node.py

```diff
@@ -438,7 +438,7 @@ def run_node(self, check: bool = False) -> float:
 
         # Execute notebook without problems:
         try:
-            self.output_parameters = self._execute_notebook(self.nb_path, output_path, params)
+            self.output_parameters = self._execute_notebook(os.path.abspath(self.nb_path), output_path, params)
```

Comment:
Why son't we transform the `ouput_path` to absolute

Reply:
I see you do that already in the function `self._create_notebook_datetime_path(dirty=True)`, maybe add it to the documentation, so Its more clear? 🙌

Reply:
And the reason we can't simply do:
`self.output_parameters = self._execute_notebook(os.path.abspath(self.nb_path), output_path, params)`
is because we get the joins weird like this
![image](https://github.com/qilimanjaro-tech/qililab/assets/109400222/ecf1617d-bbcd-41df-a4bc-0a3c99638160)

Reply:
Still, seems a bit ugly, that we do this absolute conversion here in the middle, what happens with the other calls of the function `_create_notebook_datetime_path()`

Do the fact that this function now returns an absolute path, work okei with the other calls, why?

Reply:
It looks much better and simpler to follow on the init, just changed ir!